src/mappings/escrow.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable prefer-const */
+import { BigInt, BigDecimal } from '@graphprotocol/graph-ts'
+import {
+    Transaction,
+    Sender,
+    Receiver,
+    EscrowAccount,
+    AuthorizedSigner,
+} from '../types/schema'
+import { Deposit, Withdraw, Redeem, Thaw, AuthorizeSigner, RevokeAuthorizedSigner} from '../types/Escrow/Escrow'
+let ZERO_BD = BigDecimal.fromString('0')
+let ZERO_BI = BigInt.fromI32(0)
+
+export function handleThaw(event: Thaw): void {
+    let sender_receiver = event.params.sender.toHexString() + event.params.receiver.toHexString()
```

Comment:
Could this be something like `event.params.sender.toHexString() + '-' + event.params.receiver.toHexString()`? Just to make it a bit prettier.